### PR TITLE
saas_portal: Simple Error handling for request

### PR DESCRIPTION
**Pull request by tarzan0820**
Block to enable simple error handling for calls to request.get

Helps present helpful request error message that can easily be acted on...
For instance saas_portal/models/saas_portal.py line 98 directly sends the result of the request to 
simplejson.loads which spits out errors msgs that can have you chasing your tail for awhile.

Anywaz what are thoughts on this

### CODE REVIEW

**Comment by yelizariev** [comment]
ok, I think it makes sense.
Could you update your fork, so this PR could be automatically merged

thank you

**Comment by tarzan0820** [comment]
Done

File: saas_portal/models/wizard.py [hunk 1]
```diff
@@ -61,7 +61,13 @@ def upgrade_database(self):
...
         )[0]
         res = requests.get(url, verify=(self.server_id.request_scheme == 'https' and self.server_id.verify_ssl))
-        obj.write({'description': res.text})
+    	if res.ok != True:
+    	    msg = """Status Code - %s 
+    	        Reason - %s
+    	        URL - %s
+    	        """ % (res.status_code, res.reason, res.url)
+    	    raise Warning(msg)
+    	obj.write({'description': res.text})
```
Comment (yelizariev) [inline saas_portal/models/wizard.py:70]: Be sure, that you don't use tabs

File: saas_portal/models/saas_portal.py [hunk 2]
```diff
@@ -356,7 +362,7 @@ def _delete_database_server(self, force_delete=False):
         if force_delete:
             state['force_delete'] = 1
         url = self.server_id._request_server(path='/saas_server/delete_database', state=state, client_id=self.client_id)[0]
-        res = requests.get(url, verify=(self.server_id.request_scheme == 'https' and self.server_id.verify_ssl))
+        res = requests.get(url)
```
Comment (yelizariev) [inline saas_portal/models/saas_portal.py:365]: it removes your previous update

**Comment by yelizariev** [comment]
@tarzan0820 
I'm about to rename branch from master to 8.0
Could fix issues I add on comment and create new PR for 8.0 branch

**Pull request closed without merging**